lesson06/assignment/css/style.scss

```diff
@@ -3,4 +3,9 @@
 @import 'settings.variables';
 @import 'settings.responsive';
 @import 'components.content';
-
+img{
+  width: 50%;
+}
+.bg-image{
+  background-image: url(https://mdbootstrap.com/img/Photos/Others/gradient1.jpg);
+}
```

Comment:
This assignment requires styling with CSS Grid and Flexbox. I'm not seeing any code that does that.